include/votca/tools/calculator.h

```diff
@@ -81,26 +81,28 @@ class Calculator {
   /**
    * \brief Loads default options stored in VOTCASHARE
    */
-  void LoadDefaults(std::string package = "tools");
+  Property LoadDefaults(std::string package = "tools");
 
   /**
-   * \brief Updates options with default options stored in VOTCASHARE
+   * \brief Updates user options with default options stored in VOTCASHARE
    *
    * If a value is not given or tag is not present and at the same time
    * a default value exists in the corresponding XML file in VOTCASHARE
    * a tag is created and/or a default value is assigned to it
    */
-  void UpdateWithUserOptions(const Property &user_options);
+  void UpdateWithUserOptions(Property &default_options,
+                                 const Property &user_options);
 
   /**
    * \brief Load the default options and merge them with the user input
    *
    * Defaults are overwritten with user input
    */
-  void LoadDefaultsAndUpdateWithUserOptions(std::string package,
-                                            const Property &options) {
-    LoadDefaults(package);
-    UpdateWithUserOptions(options);
+  Property LoadDefaultsAndUpdateWithUserOptions(std::string package,
+                                                const Property &options) {
```

Comment:
can you please rename the input Property options is not a good name?

Reply:
is this const?

Reply:
I have renamed it to `user_options`

Reply:
the function i.e 

` LoadDefaultsAndUpdateWithUserOptions(std::string package,
                                                const Property &options) const` ?